Resources/Prototypes/_LostParadise/Catalog/Fills/Lockers/suit_storage.yml

```
@@ -0,0 +1,71 @@
+        - id: ClothingShoesBootsMag
+        - id: ClothingOuterHardsuitSalvage
+        - id: ClothingMaskGasExplorer
+        - id: JetpackMiniFilled
+  - type: AccessReader
+
+- type: entity
+  id: SuitStorageQuartermaster
+  parent: SuitStorageBase
+  suffix: Quartermaster
+  components:
+  - type: StorageFill
```

yamllint (1.35.1)

[error] wrong indentation: expected 6 but found 2

(indentation)